aten/src/TH/generic/THTensor.cpp

```diff
@@ -217,6 +218,7 @@ void THTensor_(resize)(THTensor *self, at::IntArrayRef size, at::IntArrayRef str
 
 void THTensor_(resizeAs)(THTensor *self, THTensor *src)
 {
+  // already available in Aten as at::resize_as_()
```

Comment:
why not call resize_as_ here?

Reply:
Well, because I had no reason to change this function as we going to nuke it all together as call sites goes away.